source/auth/auth.rst

```diff
@@ -620,6 +632,18 @@ mechanism
 mechanism_properties
 	MUST NOT be specified.
 
+Caching credentials
```

Comment:
I don't think you need this in two places. Just put the text in one place titled "Caching credentials for SCRAM mechanisms" or something similar.

Reply:
I agree.  A separate "Caching SCRAM credentials" section makes sense -- but be sure to distinguish what passwords are relevant -- i.e. the mongodb hashed one (SHA-1) or the SASLprepped one (SHA-2).